libs/designer/src/lib/core/utils/parameters/helper.ts

```diff
@@ -494,44 +495,39 @@ export const parseArrayItemSchema = (itemSchema: any, itemPath = ''): any => {
 };
 
 // Create SimpleQueryBuilder Editor View Model
-const toSimpleQueryBuilderViewModel = (input: any): { isOldFormat: boolean; items: RowItemProps } => {
-  let operand1: ValueSegment[], operand2: ValueSegment[], operation: string;
+const toSimpleQueryBuilderViewModel = (
+  input: any
+): { isOldFormat: boolean; itemValue: ValueSegment[] | undefined; isRowFormat: boolean } => {
+  const advancedModeResult = { isOldFormat: true, isRowFormat: false, itemValue: undefined };
+  let operand1: ValueSegment, operand2: ValueSegment, operationLiteral: ValueSegment;
+  const separatorLiteral: ValueSegment = { id: guid(), type: ValueSegmentType.LITERAL, value: `,` };
+  const endingLiteral: ValueSegment = { id: guid(), type: ValueSegmentType.LITERAL, value: `)` };
+  // default value
+  if (!input || input.length === 0) {
+    return { isOldFormat: true, isRowFormat: true, itemValue: [{ id: guid(), type: ValueSegmentType.LITERAL, value: "@equals('','')" }] };
```

Comment:
Why do we need a default value here and why can't we use itemValue: parameterValue which has value ''. Shouldn't the editor have handling based on that?